build/filter.go

```diff
@@ -32,6 +32,21 @@ type HostFilter interface {
 type Controller0Filter struct {
 }
 
+func CleanEmptyAttributes(deployment *Deployment) {
```

Comment:
Suggest to rename it to CleanEmptyControllerAddresses. I'm wondering if the gateway address is null, will it be dumped out as ""?

Reply:
Gateway is working as expected
I have verified it

Reply:
![image](https://github.com/user-attachments/assets/87e1ee6c-07ba-4b33-970f-5699ea2bc19b)
![image](https://github.com/user-attachments/assets/286d24f9-55f4-4dbd-b6e4-b73b5e82a3ed)